packages/apollo-server-graphql-api/server/app.js

```diff
@@ -0,0 +1,15 @@
+const { ApolloServer } = require("apollo-server");
+const { readFileSync } = require("fs");
+const resolvers = require("./resolvers");
+const typeDefs = readFileSync(`${__dirname}/typeDefs.graphql`, "utf-8");
+const db = require("./db");
+
+const context = { db };
+
+const server = new ApolloServer({
+    resolvers,
+    typeDefs,
+    context,
```

Comment:
Is this final comma added by prettier?
Otherwise please remove those final commas.

Reply:
Yes, it's added by prettier